services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/tormaystarkastelu/TormaystarkasteluTulos.kt

```diff
@@ -0,0 +1,22 @@
+package fi.hel.haitaton.hanke.tormaystarkastelu
+
+class TormaystarkasteluTulos {
+    var hankeId : Int = 0
+    var liikennehaittaIndeksi: LiikennehaittaIndeksiType? = null
+    var perusIndeksi: Double? = null
```

Comment:
Do these really need to be Doubles (64-bit) instead of Float (32-bit). Even Floats have about 6 digits of precision (and we're only using less than two).  Not that there is much of a penalty using Doubles, as there amount of data and calculations are quite low, but using Floats would be a better match for the use case.

Reply:
Changed to float